decidim-participatory_processes/lib/decidim/participatory_processes/admin_engine.rb

```diff
@@ -150,6 +150,26 @@ class AdminEngine < ::Rails::Engine
 
       initializer "decidim_participatory_processes_admin.process_components_menu" do
         Decidim.menu :admin_participatory_process_components_menu do |menu|
+          current_participatory_space.components.each do |component|
+            caption = translated_attribute(component.name)
+            if component.primary_stat.present?
+              caption += content_tag(:span, component.primary_stat, class: component.primary_stat.zero? ? "component-counter component-counter--off" : "component-counter")
+            end
+
+            menu.add_item [component.manifest_name, component.id].join("_"),
+                          caption.html_safe,
+                          manage_component_path(component),
+                          active: is_active_link?(manage_component_path(component)) ||
+                                  is_active_link?(decidim_admin_assemblies.edit_component_path(current_participatory_space, component)) ||
```

Comment:
I don't like referencing cross components. Could we change this one out? Also if we need to do it, then we should move it to another that's more transversal, such as `decidim-admin`.

Finally, if we need to do so, we should also check that this module is installed, as an installation could not have it (using the `Decidim.module_installed?`  method)

Reply:
This is a bug, and was fixed in a8890b7d4e